src/bin/add/args.rs

```diff
@@ -1,4 +1,4 @@
-///! Handle `cargo add` arguments
+/// ! Handle `cargo add` arguments
```

Comment:
It's not important, but if you have the time, it would be great if you could change it to start with `//!` :)